src/test/java/io/openliberty/tools/intellij/it/TestUtils.java

```diff
@@ -279,6 +279,14 @@ public static void validateStanzaInServerXML(String pathToServerXml, String inse
         }
     }
 
+    public static void validateConfigStringInConfigFile(String pathToConfigFile, String expectedServerEnvString) {
```

Comment:
recommend renaming `expectedServerEnvString` to something like `expectedConfigString`